src/test/java/test/SimpleBaseTest.java

```diff
@@ -345,7 +347,19 @@ protected static File createDirInTempDir(String dir) {
    */
   protected static List<Integer> grep(File fileName, String regexp, List<String> resultLines) {
     List<Integer> resultLineNumbers = new ArrayList<>();
-    try (BufferedReader fr = new BufferedReader(new FileReader(fileName))) {
+    Reader reader;
+    try {
+      reader = new FileReader(fileName);
```

Comment:
The reader should be located in the try even if we catch FileNotFoundException

Reply:
Fixed, amended commit and pushed.